docs/DESIGN.md

```diff
@@ -0,0 +1,34 @@
+# Patchwork's Design Principles
+
+This document describes the design goals and philosophy of Patchwork. Features of and changes to Patchwork should always aim to follow these principles. If we really want to add a feature/make a change to Patchwork that goes against these principles, we should think about changing the principles first, rather than ignoring them and making the change anyway.
+
+This document should be version tracked, and any change to it should go through the [advice process](http://www.reinventingorganizationswiki.com/Decision_Making) with the stakeholders listed below.
+
+### Stakeholders
+* Patchwork contributors
+* Patchwork users
+* The broader Scuttlebutt community
+* People outside the Scuttlebutt community who might be affected
+
+## Design goals
+
+### Goals and scope: Patchwork should...
+* Follow the [Scuttlebutt principles](https://www.scuttlebutt.nz/principles/)
+* Be a client that people can feel confident recommending to their friends who aren't using Scuttlebutt
+* Be safe (privacy, blocking, etc)
+* Be usable (not confusing or broken; clear explanations of new concepts)
+* Be healthy (non-addictive, calm, consentful)
+* Be accessible (to the extent practical -- some a11y features are hard to retrofit into the existing code)
+* Be a fun client to use -- users should enjoy using it and, in a positive way, be tempted to try other Scuttlebutt clients
+* Look good
+    
```

Comment:
I guess, my question is, are we going to try make this client, the defacto default/friendly client for newbies to explore ssb for now ?? In that case could we add

* Be intuitive in user experience for new scuttlebutt users and non-tech-job centric people

perhaps related to asking shall we help in things like transitioning to messages that don't rely on markdown/ related to how might we approach the Content warnings. prioritization of these things

Reply:
About the non-tech thing, could we maybe change that other line to something like this instead?:

> Be a client that people can feel confident recommending to their friends who aren't using Scuttlebutt, and who might not have much tech knowledge